### ISSUE TITLE
File full_pipeline_maMIMO_DNNEst.sh does not run

### ISSUE THREAD

**Issue by granud** [mauro-belgiovine/DL-channel-estimation-MaMIMO#3]
hello，
I trained the model with the dataset you provided, achieving a training loss of less than 0.1. (TRAIN_Npkt=9000)
![image](https://github.com/mauro-belgiovine/DL-channel-estimation-MaMIMO/assets/53365527/d8870dcf-bd26-4ed7-9173-d5fae84bcc86)
However, in the test set with an SNR of '-20' you provided, the loss skyrocketed to 73.14. And the blue prediction curve significantly deviates from the actual curve. Is this result correct?
![49dc433fcc4c3789e06fe242fd6d847](https://github.com/mauro-belgiovine/DL-channel-estimation-MaMIMO/assets/53365527/4e310cb8-1bd8-467e-950a-20239f5973b0)
![9999](https://github.com/mauro-belgiovine/DL-channel-estimation-MaMIMO/assets/53365527/fe4694ae-c2e2-4dc9-bb49-a8551e57d58f)
Additionally, I noticed that in order to continue using your dataset, I need the test matlab files to run the program. Could you please provide these files?
![fd4d2ad069526a2739b329d8d3f3717](https://github.com/mauro-belgiovine/DL-channel-estimation-MaMIMO/assets/53365527/12cca927-79c9-49cf-85cf-f1e0b98fc93d)

please advise.
